openslides_backend/action/actions/poll/reset.py

```diff
@@ -28,6 +28,10 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         )
         instance["is_pseudoanonymized"] = poll.get("type") == Poll.TYPE_PSEUDOANONYMOUS
         instance["voted_ids"] = []
+        instance["entitled_users_at_stop"] = []
+        instance["votesvalid"] = "0.000000"
+        instance["votesinvalid"] = "0.000000"
+        instance["votescast"] = "0.000000"
```

Comment:
Although only a technicality, I think it's better to set these all to `None`, as would be the case for a newly created poll

Reply:
Set these 4 fields to None in reset and update the tests.